com.unity.render-pipelines.core/Runtime/Debugging/DebugManager.cs

```diff
@@ -86,15 +86,30 @@ public ReadOnlyCollection<DebugUI.Panel> panels
         /// </summary>
         public bool displayEditorUI => m_EditorOpen;
         /// <summary>
-        /// Controls whether the hotkey for opening and closing the debug window is enabled.
-        /// </summary>
-        public bool enableWindowHotkey = true;
```

Comment:
I assume it's OK to change public API here because this was just added a couple of weeks ago.

Reply:
yes